test/components/plots/xyPlotTests.ts

```diff
@@ -27,6 +27,24 @@ describe("Plots", () => {
           .renderTo(svg);
     });
 
+    it("autorange() getter", () => {
+      assert.deepEqual(plot.autorange(), "none");
+      assert.deepEqual(plot.autorange("x"), plot);
+      assert.deepEqual(plot.autorange(), "x");
+      plot.autorange("y");
+      assert.deepEqual(plot.autorange(), "y");
+      plot.autorange("none");
+      assert.deepEqual(plot.autorange(), "none");
```

Comment:
Should be `assert.strictEqual` here.

Also we should have a message on the asserts, so if one of them fails in the test it's easy to find which one. For example:
```Typescript
assert.strictEqual(plot.autorange("x"), plot, "autorange(arg) returns the calling Plot");
```